doc/source/index.rst

```diff
@@ -101,6 +106,35 @@ The astronomy module enables computation of certain parameters of interest for s
     >>> astronomy.sun_zenith_angle(utc_time, lon, lat)
     62.685986438071602
 
+
+.. envvar:: PYORBITAL_CONFIG_PATH
+
+   It is possible (but not mandatory) to define this environment variable to
+   have full control of certain static data used by Pyorbital:
+
+   Pyorbital comes with a file platforms.txt that maps a satellite name to the
+   NORAD identifier. This internal file is accessed by Pyorbital without the
+   user having to do anything.  But if you need to change or update this file
+   you can make your own copy and place in the directory pointed to by this
+   environment variable.
+
+.. envvar:: TLES
```

Comment:
I'd prefer if the first sentence in this description was what type of value this should have (is it a directory or a single file? does it accept multiple paths? if so, colon separated?) and if it is optional. Then I think the more detailed description can come after it.

Reply:
I think this should be better now?